src/Engine/Audio/SteamAudioIndirectRenderer.cpp

```diff
@@ -1,11 +1,13 @@
 #include "SteamAudioIndirectRenderer.hpp"
 
 SteamAudioIndirectRenderer::SteamAudioIndirectRenderer() {
-
+    environment = nullptr;
+    context = nullptr;
 }
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'SteamAudioIndirectRenderer::environment' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)